06_modules_lab/01.py

```diff
@@ -1,4 +1,14 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+
+import sys
+
+if len(sys.argv) < 2:
+	print "Error, please send number as argument"
+	sys.exit()

```

Comment:
Better to write the error message inside `sys.exit()`. It prints the error to stderr AND returns an error code to the operating system.